core/src/main/java/com/netflix/conductor/core/execution/DeciderService.java

```diff
@@ -882,6 +892,7 @@ public static class DeciderOutcome {
         List<TaskModel> tasksToBeScheduled = new LinkedList<>();
         List<TaskModel> tasksToBeUpdated = new LinkedList<>();
         boolean isComplete;
+        Optional<TaskModel> terminateTask = Optional.empty();
```

Comment:
`Optional` is typically not used as a field. https://blog.joda.org/2014/11/optional-in-java-se-8.html